.commitlintrc.json

```diff
@@ -30,7 +30,7 @@
         "rgw",
         "security",
         "subvolumegroup",
-	"namespace",
+	      "namespace",
```

Comment:
Intentional change?

Reply:
Not directly, the spacing is off and my editor fixed it. I'll create a separate PR in the future to fix this small nit.